admin/build-doc

```diff
@@ -20,7 +20,7 @@ if command -v dpkg >/dev/null; then
         exit 1
     fi
 elif command -v yum >/dev/null; then
-    for package in python-devel python-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
+    for package in python-devel python2-pip python-virtualenv doxygen ditaa ant libxml2-devel libxslt-devel Cython graphviz; do
```

Comment:
my patch is not a good solution. First, new python-pip 8.**  rpm packages name has been chagned with python2-pip becase new official spec file not mine, that's what i am got.  Second, What i want to say is we cloud not only using a rpm package name to judge program has been install，it's not one-on-one. Third. i have no good solution, i tried like : yum list installed $package -q > /dev/null 2>/dev/null; so, python*-pip will be matched.